Decentralization - Using Pledge as a Bidding Parameter to Determine Saturation Limit.md

```diff
@@ -0,0 +1,233 @@
+CIP: TBD\
+Title: Decentralization: Using Pledge as a Bidding Parameter to Determine Saturation Limit\
+Author: Jay Pseudonym Cappucino \<jycappucino@gmail.com>\
+Status: Draft\
+Type: Process\
+Created: 2022-02-28\

```

Comment:
You will have to surround the entire header section text with `---` characters (as you can see in https://raw.githubusercontent.com/cardano-foundation/CIPs/master/CIP-0001/README.md) make it format like the other CIPs with a table in the header (as you see in https://github.com/cardano-foundation/CIPs/tree/master/CIP-0001).